### DISCUSSION TITLE
"Suggest Answer", "Comment" to be ½-¾ height, not on top...

### DISCUSSION THREAD

**Discussion by unique-EJ** [community/community#13978] [Mobile]
...inside this.

![Screenshot_20220331-132122_GitHub.png](https://user-images.githubusercontent.com/91560320/161053555-332fce91-27e3-4589-a180-bae1a7480592.png)

**Comment by jacortinas** [comment]
Thank you for the feedback and screenshot, @e1e0! This definitely looks like something that we need to correct; I've created an issue to track this bug and we'll fix it in a release very soon.

**Reply by Jay-o-Way** [reply]
Is this still an issue with the latest version?

**Reply by unique-EJ** [reply]
**Yes.**
